core/food_apis/scheduler_runtime.py

```
@@ -0,0 +1,293 @@
+        _LOCAL_UPDATE_LEASE.release()
+
+
+async def run_with_update_lease(
+    operation: UpdateOperation[T],
+    *,
+    mode: SchedulerMode | None = None,
+    session_factory: SessionFactory | None = None,
+) -> T:
+    """Run one update attempt through the canonical coordination boundary."""
+
+    resolved_mode = validate_scheduler_mode(mode) if mode is not None else resolve_scheduler_mode()
```

Flake8 (7.3.0)

[error] local variable 'resolved_mode' is assigned to but never used

(F841)